src/Engines/Extensions/engine.alpha.ts

```diff
@@ -118,6 +118,10 @@ ThinEngine.prototype.setAlphaMode = function (mode: number, noDepthWriteChange:
             this._alphaState.setAlphaBlendFunctionParameters(this._gl.ONE_MINUS_DST_COLOR, this._gl.ONE_MINUS_SRC_COLOR, this._gl.ZERO, this._gl.ONE);
             this._alphaState.alphaBlend = true;
             break;
+        case Constants.ALPHA_LAYER_ACCUMULATE:
+            this._alphaState.setAlphaBlendFunctionParameters(this._gl.SRC_ALPHA, this._gl.ONE_MINUS_SRC_ALPHA, this._gl.ONE, this._gl.ONE_MINUS_SRC_ALPHA);

```

Comment:
Same comment as some others, would be great to know where this setups comes from :-)